### ISSUE TITLE
Joint Ownership

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#19]
At a recent Provenance Index meeting, we presented the top two options for modeling joint ownership -- virtually partition the object and track each separately, or create a virtual group of people who would jointly own the object.  The recommendation was that the virtual partitioning is preferable for research purposes, as it's easier to track who owned how much of what at any given time, and those partitions are sold, resold and transferred in the same way as the modern day stock market.

However, LegalObject is a superclass of PhysicalThing, which is where P46 consists_of (aka `part`) is defined and what can have its ownership transferred by E8. So we cannot have a Painting have LegalObject parts. Also, Acquisition can only transfer E18 Physical Objects.  So I don't think we can, given the current CRM constraints, actually have virtual parts of objects be owned.

Given the current discussion about Rights, maybe we've caught sufficient interest from the SIG to result in some improvements to the model here.

**Comment by azaroth42** [comment]
Following the alternative Group method, to demonstrate that we cannot do that either:

The ownership is transferred to a Group that consists of the set of owners. We would need a way to associate the degree of membership in the group (e.g. 25% / 75%).  This is important as the membership of the group might not change with transactions e.g. the 75% holder sells another 25% to the first, resulting in the same two members, but now 50/50.  This could be treated as a different group, but we still lack the degree.

Groups have current or former members, which can have a Type (though not in RDF). However the type is also not able to help, as we would need an infinite number of types (half-owner, quarter-owner, eighth-owner, ...). 

CRM thus seems to lack any way to model this, and we should consider how best to integrate.

**Comment by workergnome** [comment]
This is my first stab at this, based on the Group method, which likely has problems:

```
_:sale transferred_title_of _:object;
    transferred_title_to _:consortium. 

_:consortium a crm:E40_Legal_Body; 
    crm:P107_has_current_or_former_member _:Larry;
    crm:P107_has_current_or_former_member _:Curly;
    crm:P105i_has_right_on _:partnership_agreement.

_:partnership_agreement a crm:E72_Legal_Object;
    crm:P104i_applies_to _:object;
    crm:P46_is_composed_of _:share1, _:share2, .

_:share1 crm:E72_Legal_Object ;
  label “50% share of Painting” ;
  crm:P105_right_held_by _:Larry;
  has_dimension _:fiftyPercent .

_:share2 crm:E72_Legal_Object ;
  label “50% share of Painting” ;
  crm:P105_right_held_by _:Moe;
  has_dimension _:fiftyPercent .

_:fiftyPercent a Dimension ;
  value 0.5 ;
  unit [ a Unit ; p2_has_type aat:ownership ] .
  ```

**Comment by azaroth42** [comment]
I think the use of 148 here is possible due to the weirdness of E30 being a Propositional Object.  Martin said on the SIG list that it was intended to be a particular expression of the right:

> E30 is intended to be the content of the right, which is relatively trivial if it is just an instance of unspecified copyright, but not if it is an individual contract.

So I would interpret the P148 as the expression of the right has two parts -- like the main part and then the riders added on -- rather than the abstraction it represents is split.  Seems like a good test case for the SIG though?

**Comment by azaroth42** [comment]
azaroth42 marks this issue as being discussed in [slack](https://linked-art.slackarchive.io/-/search-24d7e6d4f2c54ad4a08d1ab8ee8ae102/page-1)

**Comment by azaroth42** [comment]
Progress towards this issue:  https://multiple-owners--linked-art.netlify.com/model/provenance/multiple/

**Comment by azaroth42** [comment]
@workergnome and I have come up with a revised model that accommodates copyright, custody and other sorts of legal claims, jurisdiction, theft/looting as well as partial ownership, following on from the discussions with the SIG about the implication of E30 Right being a Conceptual Object that hence cannot be destroyed.  It also works for the Art Tracks patterns (as far as we thought through them)

To summarize the model:

* A LegalClaim instantiates a Right.  Rights (per the SIG) are conceptual and indefinite, thus not something that can be created or destroyed.  The separation of Claim from the Right fixes that.
* Claims can be brought into existence by a BeginningOfExistence event. That event will typically be part of an Acquisition.
* Claims can be split into parts, that have Dimensions. The dimension for each part lets us have arbitrary splits of claimed right (e.g. 30% ownership)
* Claims are claimed by Actors (people or organizations). We don't think there is a *need* for a Group to be created to capture the set of actors that jointly share the right, simply create multiple relationships from the claim to the different actors.
* Claims have a jurisdiction of a Place in which they apply, meaning that a copyright claim can be scoped to North America and not in Europe.
* Claims can be classified as either legally valid or illegal
* The Right is necessary to determine what sort of right is being claimed, and hence the Right will have a Type to distinguish (e.g.) ownership from copyright from custody from reproduction.
* Any change of details for the claim results in the E81 Transformation of the previous claim into a new one, including acquisitions or just changes to partial gifts from (e.g.) 60/40 to 70/30.
* The Transformation will have one or two of resulted_in and transformed ... depending on whether the previous/future claim is also known. If not, no need to create artificial ones.
* The Production of the object brings a Claim into existence of the artist over the object.
* The Destruction/EoE of the object ends the existence of the current Claim.

[image upload seems broken at the moment, imagine a nice omnigraffle diagram here :)]

**Comment by azaroth42** [comment]
Further discussion:
* There is exactly one Right per classification per object, and thus must not have the possessed_by relationship (in our understanding of "Right" as a Conceptual Object). 

There is an outstanding question of how best to associate temporally spanning activities with state-changing activities, that we've taken to the SIG in the first instance.  This especially affects how the Transformation and Payment are aligned for the ongoing transfer of partial ownership between parties -- there isn't a new Acquisition (as they apply to the whole object) but as money and shares-of-value change hands, the state changes need to be linked together somehow.

**Comment by azaroth42** [comment]
Closed by #122
